apps/researcher/src/lib/community.ts

```diff
@@ -28,8 +31,58 @@ export async function getMemberships(
   });
 }
 
-export function getAllCommunities(): Promise<Community[]> {
-  return clerkClient.organizations.getOrganizationList();
+export enum CommunitySortBy {
+  NameAsc = 'nameAsc',
+  NameDesc = 'nameDesc',
+  CreatedAtDesc = 'createdAtDesc',
+  MembershipCountDesc = 'membershipCountDesc',
+}
+
+export const defaultCommunitySortBy: CommunitySortBy =
```

Comment:
Curious: is the explicit type `CommunitySortBy` required by TypeScript? Or can we omit it?